arrow-libs/core/arrow-core/src/commonTest/kotlin/arrow/core/raise/IorSpec.kt

```diff
@@ -79,4 +75,13 @@ class IorSpec : StringSpec({
       }
     }.message shouldBe "Boom!"
   }
+
+  "Recover works as expected" {
```

Comment:
This highlights the problem in the issue. This test fails without the change above.